internal/provider/devHTTP.go

```diff
@@ -5,7 +5,7 @@ import (
 	"fmt"
 	"net/http"
 	"strings"
-	"github.com/baserrato/devops-resource"
+	"github.com/liatrio/devops-bootcamp/examples/ch6/devops-resources"
```

Comment:
did this decide to work?

Reply:
Yes it finally did. It's enormous but it will work in keeping provided modules to be used by custom providers